code/controllers/subsystem/voting.dm

```diff
@@ -116,7 +116,18 @@ var/datum/subsystem/vote/SSvote
 					else
 						master_mode = .
 	if(restart)
-		world.Reboot("Restart vote successful.", "end_error", "restart vote")
+		if(!admins.len)
+			return
+		var/total_admins
+		var/afk_admins
+		for(var/client/C in admins)
+			total_admins++
+			if(C.is_afk())
+				afk_admins++
+		var/active_admins
+		active_admins = total_admins - afk_admins
+		if(active_admins <= 0)
+			world.Reboot("Restart vote successful.", "end_error", "restart vote")

```

Comment:
This needs an else, to let the users know that it is waiting on the admins.

As well as let the admins know that they need to decide to restart.

This way admins can allow a restart vote, and if it passes, decide to restart the server, AND users know this is going on.